.eslintrc.js

```diff
@@ -1,3 +1,20 @@
+const globalRestrictedImport = [
+    {
+        name: 'react-native',
+        importNames: ['useWindowDimensions', 'TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight', 'StatusBar'],
+        message: [
+            "For 'useWindowDimensions', please use 'src/hooks/useWindowDimensions' instead.",
+            "For 'TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight', please use 'src/components/Pressable' instead.",
```

Comment:
should we be more specific on which component from `Pressable` to use in the message like the message for `react-native-gesture-handler`

```
Please use 'PressableWithFeedback' and/or 'PressableWithoutFeedback' from 'src/components/Pressable' instead
```